README.md

```diff
@@ -1,5 +1,23 @@
 # MakersBnB
 
+## Instructions for using the app
+
+- Start by cloning this repo
+- Run bundler to install dependencies. Follow the instructions to install any that are missing and not automatically installed. 
+- set up your tables as follows:
+- enter ```# psql postgresql``` in the terminal:
+- You should now see something like the following:<br>
+ ```psql (14.4)```<br>
+ ```Type "help" for help.```
+- once in psql, enter the info found in db/migrations/01_create_databases.sql (as individual entries). This will create the databases
+- then navigate to each database with ```# \c makersbnb``` and ```# \c makersbnb_test```, and once inside each one, enter all the info found in db/migrations/02_create_tables.sql (as individual entries). This will create the tables for the user experience and for testing. The information should be the same in both databases.  
+- start up the server with ```$ rackup config.ru``` or ```$ rackup -p 3000```
+- navigate to the relevant localhost address, either ```localhost:9292``` or ```localhost:3000```, depending on the above choice
+- to enter information, follow the instructions on the page
+- to view the info in the databases, navigate to the you wish to view database, then enter ```# \dt``` to view all tables.
```

Comment:
Didn't understand this part. Do you mean on Table Plus or something like that?

Reply:
I meant using psql. I can simplify it and resubmit